app/qml/main.qml

```diff
@@ -61,6 +61,24 @@ ApplicationWindow {
         tab.focus()
     }
 
+    function gotoRightTab() {
+        console.log("Going to right tab...")
+        var newIndex = (tabbedPage.currentIndex + 1) % (tabbedPage.count)
+        tabbedPage.setCurrentIndex(newIndex)
+    }
+
+    function gotoLeftTab() {
+        console.log("Going to left tab...")
+        var newIndex
+        if(tabbedPage.currentIndex == 0) {
+            newIndex = tabbedPage.count - 1
+        }
+        else {
+            newIndex = (tabbedPage.currentIndex -  1) % (tabbedPage.count)
+        }
```

Comment:
nitpick: code style is different, it should be:

```
if (condition)
   one_line_of_code();
else
    one_line_of_code();
```

or in case of ifs with multiple lines of code:

```
if (condition) {
    one_line():
    two_line();
} else {
    even_if_else_has_only_one_line_of_code();
}
```

Reply:
Solved on commit 4fd64f4